.github/workflows/install-script.yml

```diff
@@ -0,0 +1,47 @@
+name: Install Script
+on:
+  push:
+    paths:
+      - "install/**"
+  pull_request:
+    paths:
+      - "install/**"
+
+jobs:
+  shellcheck:
+    name: Shellcheck [Linter]
+    runs-on: ubuntu-latest
+    permissions:
+      checks: write
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Setup | Checkout
+        uses: actions/checkout@v6
+
+      - name: Lint | Run shellcheck
+        uses: reviewdog/action-shellcheck@v1
```

Comment:
I think I would prefer just running the CLI-tool here, it should already be installed on the ubuntu runner. Something like `shfmt` would also be interesting.